general-concepts/manifest/text.xml

```diff
@@ -15,9 +15,10 @@ by the package. This is used to verify integrity upon fetching them.
 </p>
 
 <p>
-To generate the <c>Manifest</c>, use <c>ebuild foo.ebuild manifest</c>
-or <c>repoman manifest</c>. You may want to set <c>GENTOO_MIRRORS=</c> while
-calling it to fetch distfiles from their original locations immediately.
+To generate the <c>Manifest</c>, use <c>ebuild foo.ebuild manifest</c>,
+<c>repoman manifest</c>, or <c>pkgdev manifest</c>. You may want to set
+<c>GENTOO_MIRRORS=</c> while calling it to fetch distfiles from their original
```

Comment:
`pkgdev manifest` doesn't use Gentoo mirrors by default, you have to enable the -m/--mirrors flag so it does.